www-apps/radicale/radicale-2.1.11-r2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python3_6 python3_7 )
```

Comment:
maybe change to a cleaner format like `python3_{6,7}`

optional

Reply:
563ebcb0cc1f4207ae53f70ac054ed73e687bbcf; I don’t mind either way (I find the braced format neither easier nor harder to read personally).